projects/step-frontend/src/lib/modules/execution/components/schedule-overview/schedule-overview.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, inject, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AugmentedSchedulerService, ExecutiontTaskParameters, Plan } from '@exense/step-core';
+
+@Component({
+  selector: 'step-schedule-overview',
+  templateUrl: './schedule-overview.component.html',
+  styleUrls: ['./schedule-overview.component.scss'],
+})
+export class ScheduleOverviewComponent implements OnInit {
+  private _scheduleApi = inject(AugmentedSchedulerService);
+  private taskId?: string;
+  private _taskData?: ExecutiontTaskParameters;
+
+  protected task = this._taskData;
+
+  protected plan?: Partial<Plan>;
+
+  protected error = '';
+  protected repositoryId?: string;
+  protected repositoryPlanId?: string;
+
+  constructor(private route: ActivatedRoute) {}
+
+  ngOnInit(): void {
+    this.route.url.subscribe((segments) => {
+      segments[segments.length - 1].path;
+      this.taskId = segments[segments.length - 1].path;
+      if (this.taskId) {
+        this._scheduleApi.getExecutionTaskById(this.taskId).subscribe({
+          next: (task) => {
+            this.task = task;
+            this.initializeTask();
+          },
+          error: () => {
+            this.error = 'Invalid Task Id or server error.';
+          },
+        });
+      } else {
+        this.error = 'Task Id not found in the URL.';
+      }
+    });
```

Comment:
Please avoid `subscribe` inside `subscribe`.
Refactor this observable chain with usage of `switchMap` operator.
Also `route.url` is hot observable - it's need to be unsubscribed, when component is destroyed. It's can be done with `takeUntilDestroyed` operator